Fix container cleanup process failing with error `AttributeError: 'DockerKernel' object has no attribute 'network_driver'`
